x/auth/client/rest/query.go

```diff
@@ -146,6 +152,8 @@ func QueryTxRequestHandlerFn(clientCtx client.Context) http.HandlerFunc {
 
 func queryParamsHandler(clientCtx client.Context) http.HandlerFunc {
 	return func(w http.ResponseWriter, r *http.Request) {
+		client.AddDeprecationHeaders(w)
```

Comment:
This route (`/auth/params`) is not documented in Swagger docs, so calling out explictly here. Is this endpoint considered legacy REST?